lib/apollo-federation/schema.rb

```diff
@@ -22,7 +22,7 @@ def federation(version: '1.0', link: {})
       end
 
       def federation_version
-        @federation_version || '1.0'
+        @federation_version || (superclass.respond_to?(:federation_version) && superclass.federation_version) || '1.0'
```

Comment:
Would you mind updating this implementation to use `GraphQL::Schema::FindInheritedValue`?